src/coreclr/interpreter/compiler.h

```diff
@@ -94,6 +94,7 @@ struct InterpInst
 
     int32_t opcode;
     int32_t ilOffset;
+    int32_t nativeOffset;
```

Comment:
The `InterpInst` should be as light weight as possible. As of this change I don't see us needing it, maybe it would be necessary in the future ? @kg